### ISSUE TITLE
Custom Font: message is not displayed completely when using custom font for long messages

### ISSUE THREAD

**Issue by qaisershehzad** [MessageKit/MessageKit#639]
## General Information

* iOS Version(s): 10+

* Swift Version: 4

* Reproducible in ChatExample? (Yes/No): yes, attached screenshot

##  What happened?

In Message Cell Attributed text is truncated not completely loading for long messages. Attached reference screenshot from Chat Example.
![customfont](https://user-images.githubusercontent.com/11673425/38516340-58bb9a08-3c50-11e8-8001-cc44c822124e.png)

## What did you expect to happen?

It should load complete message like loading for system fonts.

**Comment by SD10** [comment]
@qaisershehzad What is the size of your message in characters?

**Comment by qaisershehzad** [comment]
I tried on multiple strings of different lengths.
Above screenshot String length is "1614"

Length: "348"
![smallstring](https://user-images.githubusercontent.com/11673425/38534732-3753633e-3c99-11e8-9e65-64da9e0b145b.png)

Length: "182"
![182](https://user-images.githubusercontent.com/11673425/38534810-9fded258-3c99-11e8-9f83-73beb1d1ee5a.png)

**Comment by SD10** [comment]
@qaisershehzad Show me the code where you create the message, please

**Comment by qaisershehzad** [comment]
thanks for your quick response. 

I forked and pushed code with custom font.
I am changing font here https://github.com/qaisershehzad/MessageKit/blob/master/Example/Sources/SampleData.swift#L85-L86.
If you uncomment line 86 and comment line 85 then it start truncating text.
like below screemshot

![customfont_colfax](https://user-images.githubusercontent.com/11673425/38537742-bc63e0ca-3ca9-11e8-9c03-682aab37e484.png)

But if you comment line 86 
![systemfont](https://user-images.githubusercontent.com/11673425/38537755-d744cc4c-3ca9-11e8-8946-ffc2db39aadf.png)

In my findings the height returned by https://github.com/qaisershehzad/MessageKit/blob/master/Sources/Extensions/NSAttributedString%2BExtensions.swift#L33 is incorrect for custom fonts.

**Comment by SD10** [comment]
Thanks for following up @qaisershehzad. I'm not sure how to handle this one because I'm relying on `Foundation` to size attributed strings properly. I'm going to say it's lower priority right now considering this is regarding a custom font. If you are willing to dig deeper on this one please let us know what you find

**Comment by zhongwuzw** [comment]
I will investigate this a while later.

**Comment by qaisershehzad** [comment]
I investigated this a bit. 

Actually the height returned by  extension is fine for custom font as well I tried on other sample project. When we give same width it return same height in other sample project and AttributedText is also completely shown on UILabel.
https://github.com/qaisershehzad/MessageKit/blob/master/Sources/Extensions/NSAttributedString%2BExtensions.swift#L33

Further I tried to disable message styling in ChatExample and a new Label is added in textMessageCell of same frame like MessageLabel 
here https://github.com/qaisershehzad/MessageKit/blob/master/Sources/Views/Cells/TextMessageCell.swift#L79-L103
So on this label text is fine as below screenshot.
![newlabel](https://user-images.githubusercontent.com/11673425/38600140-81bd0fb8-3d7c-11e8-94e7-2baa3b6b5ada.png)

So it seems issue is with MessageLabel configurations  (text insets and framing of messageContainerView and styling).
